### PR TITLE
Added new rules for aws terraform 4.0

### PR DESCRIPTION
**Pull request by harshilswan**

### CODE REVIEW

File: aws/terraform/storage.rego [hunk 1]
```diff
@@ -1210,6 +1309,45 @@ s3_website_metadata := {
...
+s3_website_v4 = false {
+    aws_issue["s3_website_v4"]
+}
+
+s3_website_v4_err = "S3 buckets with configurations set to host websites" {
+    aws_issue["s3_website_v4"]
+}
+
+s3_website_v4_metadata := {
+    "Policy Code": "PR-AWS-TRF-V4-S3-013",
```
Comment (farchide) [inline aws/terraform/storage.rego:1339]: @harshilswan naming convention should be the same.
do not add `v4` to the code
you can use `13v4`

File: aws/terraform/storage.rego [hunk 2]
```diff
@@ -1210,6 +1309,45 @@ s3_website_metadata := {
     "Resource Help URL": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html"
 }
 
+
+#
+# PR-AWS-TRF-V4-S3-013
+#
+
+default s3_website_v4 = null
```
Comment (farchide) [inline aws/terraform/storage.rego:1317]: @harshilswan 
how do we know these testcase are only getting applied to v4 of terraform aws provider?
should we check the provider version as well?

File: aws/terraform/storage.rego (hunk 1 above)
Comment (farchide) [reply aws/terraform/storage.rego:1339]: or `s3v4`

Comment (harshilswan) [reply aws/terraform/storage.rego:1339]: ok

**Review by farchide — APPROVED** [review]

**Pull request merged**